tests/scoreboard/MultiRun/run_test.pl

```diff
@@ -0,0 +1,75 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+  & eval 'exec perl -S $0 $argv:q'
+  if 0;
+
+use strict;
+use warnings;
+
+use FindBin;
+use constant autobuild_root => "$FindBin::Bin/../../../";
+$ENV{'PATH'} = "$ENV{'PATH'}:" . autobuild_root;
+use lib autobuild_root;
+chdir ($FindBin::Bin);
+
+use common::utility;
+use common::test_utils;
+
+use File::Path qw(make_path remove_tree);
+
+sub run_cmd {
+  my $cmd = shift;
+  print "$cmd\n";
+  return system($cmd);
+}
+
+sub copy_dir {
+  my $src_dir = shift;
+  my $des_dir = shift;
+  run_cmd("cp -R $src_dir $des_dir");
+}
+
+sub run_scoreboard {
+  run_cmd("../../../scoreboard.pl -c -f ./test.xml -o test.html -d runs/run");
+}
+
+sub diff {
+  my $run_a = shift;
+  my $run_b = shift;
+  my $o = shift;
+  my $r = run_cmd("diff runs/$run_a/$o runs/$run_b/$o");
```

Comment:
This also is a command I'd like to avoid calling. Since this is just seeing if they're the same this could easily be done in Perl by taking `expect_file_contents` and making a new function that compare files instead of files and a string.

Reply:
replaced with compare_files()